cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -473,6 +473,7 @@ def delete_vm_and_wait_for_gone():
 
 
 # Not collected for EC2 in generate_tests above
+@pytest.mark.meta(blockers=[BZ(1746931)])
```

Comment:
@jaryn I see that his BZ is related to Openstack Catalog Items, but not provisioning? Is provisioning also broken by this BZ?

Reply:
yes. there is a dup BZ to this I reported: 1748330

Reply:
Got it, thanks!